index.html

```diff
@@ -1012,6 +1019,144 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `file_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest">`file_handlers`</dfn></code> member is a [=list=] that
+          provides instructions for how the app handles file-opening actions
+          that originate outside of the app itself.
+        </p>
+        <p>
+          How file handlers are registered and disambiguated is at the
+          discretion of the operating system.
```

Comment:
It's not clear to me what this sentence is intended to convey. I guess it means that control over which app is used to open a file is an OS-level setting, not a user agent setting? If that's the case, maybe something like:

> The management, presentation and selection of registered file-handling applications is at the discretion of the operating system.